src/main/resources/bootstrap.yaml

```diff
@@ -12,11 +12,10 @@ spring:
         idam-sscs-oauth2-client-secret: idam.oauth2.client.secret
         idam-oauth-user: idam.oauth2.client.id
         idam-redirect: idam.oauth2.redirectUrl
-        idam-s2s-api: idam.s2s-auth.url
-        sscs-s2s-secret: idam.s2s-auth.totp_secret
-        s2s-micro: idam.s2s-auth.microservice
+        sscs-s2s-secret: IDAM_S2S_AUTH_TOTP_SECRET
+        s2s-micro: IDAM_S2S_AUTH_MICROSERVICE
```

Comment:
I would remove this, it'll always be sscs i assume?

Reply:
Addressed